sdk/src/main/java/io/opentelemetry/sdk/trace/export/SimpleSpanProcessor.java

```diff
@@ -71,15 +74,25 @@ public boolean isStartRequired() {
   }
 
   @Override
+  @SuppressWarnings("BooleanParameter")
```

Comment:
I don't think this is needed on this method.

Reply:
Hangover from the past. Removed.